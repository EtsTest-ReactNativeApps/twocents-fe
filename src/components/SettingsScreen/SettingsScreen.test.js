import React from 'react';
import { shallow } from 'enzyme';
import { SettingsScreen } from './SettingsScreen';

describe('SettingsScreen', () => {

  it('should match to snapshot', () => {
    const component = shallow(<SettingsScreen/>)
    expect(component).toMatchSnapshot()
  });
  
});