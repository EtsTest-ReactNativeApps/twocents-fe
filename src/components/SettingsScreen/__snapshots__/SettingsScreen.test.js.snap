// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsScreen should match to snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#2C2540",
      "flex": 1,
      "paddingTop": 50,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#EE933F",
        "fontSize": 30,
      }
    }
  >
    Settings!
  </Text>
</View>
`;
